### ISSUE TITLE
Robot metrics

### ISSUE THREAD

**Issue by znarf** [access-watch/access-watch#29]
Why not add robot.id to dashboard requestMetrics?

We would then be able to search which robot is responsible for a peak when zooming.

**Comment by benfle** [comment]
I agree this is an important feature but we should be able to do this without having to switch from the metrics screen to the session screen.

Regarding the implementation I'm thinking about ways to link the "metrics" to the "sessions".

**Comment by benfle** [comment]
Now that we have the speed per hour for the last 24 hours, does it fix the problem?

**Comment by znarf** [comment]
So, the use case is:

1. I have a traffic peak in my dashboard metrics
2. I want to know which agent was responsible for it
3. I'm zooming in the time interval
4. I want to list all the session/robot in that time interval

In that regard, I think the per_hour speed doesn't help solve this problem precisely enough.
